.gitignore

```diff
@@ -31,3 +31,6 @@
 /lib/tasks_private/terraform/.terraform
 /lib/tasks_private/terraform/.terraform*
 /lib/tasks_private/terraform/.byebug*
+lib/tasks_private/terraform/snapshot/terraform.*
```

Comment:
We can clean this up later, but all of these terraform ignores (include those before this commit could be combined into a few lines:

```
*.log
.terraform*
terraform.*
```

Reply:
When I did that initially, I just wanted to highlight what I am trying to ignore. That is why I made explicit statements.

Reply:
To truly make the test repeatable,  I do have a couple of changes coming along with
1. The code will remove explicit dependency on the public network name
2. If the volume is already existing, it will not recreate a volume. 
3. Start removing hard count on the VM count (and some others etc). 
These changes may be necessary as our requirements have highlighted some Issues/PRs in Terraform Plugin code.
Righjt now these changes can go as in.